codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowControllerTest.java

```diff
@@ -1050,6 +1050,10 @@ public void error(Throwable t) {
         }
 
         @Override
+        public void writeComplete() {
```

Comment:
@nmittler not completely, in the catch block you would have to move `payload.error()` call into the `if(!complete) `block.